srcCxx/ShapeType_main.cxx

```diff
@@ -79,17 +132,22 @@ Topic *create_topic(DomainParticipant *participant, const char *topic_name)
     Topic *topic = participant->create_topic(topic_name, type_name,
             TOPIC_QOS_DEFAULT, NULL /* listener */, STATUS_MASK_NONE);
     if ( topic == NULL ) {
-        printf("create_topic error for topic '%s'\n", topic_name);
+        logger.log_message("failed to create topic: " + std::string(topic_name), Verbosity::ERROR);
```

Comment:
@ClarkTucker I'm getting build errors here because of the `Verbosity::ERROR`:
```
ShapeType_main.cxx: In function ‘DDS::Topic* create_topic(DDS::DomainParticipant*, const char*)’:
ShapeType_main.cxx:135:82: error: ‘Verbosity’ is not a class or namespace
         logger.log_message("failed to create topic: " + std::string(topic_name), Verbosity::ERROR);
```
They are fixed if we use `ERROR` instead. See [fix_build_patch.txt](https://github.com/user-attachments/files/16494862/fix_build_patch.txt).

It [seems](https://stackoverflow.com/questions/5188554/my-enum-is-not-a-class-or-namespace) that scoped enums are not standard C++. Let me create a merge request to fix it.

Reply:
Adding "-std=c++11" to your compile flags should be sufficient...